index.html

```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
+                                        <button type="submit" class="see">See projects</button>
+                                      </div>
+                                    </ul>
+                        </div>
+                        
+                            </section>
+                            <section class="contact-me">
+                              <div class="head-section-co">
+                                <div id="head-container-co">
+                                  <h1 class="head-co">
+                                    About, <br />
+                                    Myself
+                                  </h1>
+                                  <p class="description-co">
+                                    Hello I’m a software developer! I can help you build a product,
+                                    feature or website Look through some of my work and experience! If
+                                    you like what you see and have a project you need coded, don’t
+                                    hestiate to contact me.
+                                  </p>
+                                  <h2 class="connect-co">LET’S CONNECT</h2>
+                        
+                                  <ul class="social-media-icons">
+                                    <li>
+                                      <a href="#" target="_blank">
+                                        <img src="images/github.png" alt="image" />
+                                      </a>
+                                    </li>
+                                    <li>
+                                      <a href="#" target="_blank">
+                                        <img src="images/Linkedin.png" alt="image" />
+                                      </a>
+                                    </li>
+                                    <li>
+                                      <a href="#" target="_blank">
+                                        <img src="images/angelite.png" alt="image" />
+                                      </a>
+                                    </li>
+                                    <li>
+                                      <a href="#" target="_blank">
+                                        <img src="images/twitter.png" alt="image" />
+                                      </a>
+                                    </li>
+                                    <li>
+                                      <a href="#" target="_blank">
+                                        <img src="images/medium.png" alt="image" />
+                                      </a>
+                                    </li>
+                                  </ul>
+                                  <div class="newmes-co">
+                                    <button type="submit" class="go-newmes">My resume</button>
+                                  </div>
```

Comment:
- [ ] Kindly remove the spaces around the about section and add space inside the section to match the figma template.

- [ ] Also, please align your social media icons horizontally to match the design 
### Your design
![3](https://user-images.githubusercontent.com/94551484/192069203-912105df-ffb8-49e6-ad56-86f8d2ea7b9a.png)
guidelines.
### Figma template
![f3](https://user-images.githubusercontent.com/94551484/192069220-4c2b82f7-04b6-4c65-95a3-3425cd25fc1c.png)